src/third-party/uploader/rusha.min.js

```diff
@@ -0,0 +1,2 @@
+/*! rusha 2017-05-05 */
```

Comment:
Why isn't this imported from node_modules?

Reply:
I'm not going to import as module since I'd like to keep the worker-specific code they have, which isn't included if you import as a module, see: https://github.com/srijs/rusha/blob/master/rusha.js#L466

Also, import as a module means the code isn't minified in the inline blob worker.